docs/resources/new_in_doc.md

```diff
@@ -7,14 +7,45 @@ month_change: true
 
 This page contains recent highlights and notable changes in [[= product_name =]] documentation.
 
+## November 2025
+
+### Releases
+
+- [v5.0.4 release notes](https://doc.ibexa.co/en/5.0/release_notes/ibexa_dxp_v5.0/#ibexa-dxp-v504)
+- [v4.6.26 release notes](https://doc.ibexa.co/en/4.6/release_notes/ibexa_dxp_v4.6/#ibexa-dxp-v4626)
+
+[MariaDB 11.4 is officially supported on v5 and v4.6](requirements.md#dbms).
+
+### Taxonomy
+
+- Described [Taxonomy suggestions](taxonomy.md#taxonomy-suggestions)
+
+### API
+
+- Expended search API to content types:
+    - [Finding and filtering content types](managing_content.md#finding-and-filtering-content-types)
+    - [Content Type Search Criteria reference](content_type_criteria.md)
+    - [Content Type Search Sort Clauses](content_type_sort_clauses.md)
+    - [Filter content types REST resource](/api/rest_api/rest_api_reference/rest_api_reference.html#tag/Type/operation/api_contenttypesview_post)
+
+### LTS Updates
+
+- Added documentation for Integrated help (v5 only):
+    - [how to install](integrated_help.md)
+    - [how to customize](customize_integrated_help.md)
+- Added [Anthropic connector installation](configure_ai_actions.md#install-anthropic-connector) (v5 only)
+- Improved Collaborative editing (v5 and v4.6):
```

Comment:
The Real-TIme configuration part is now added as well, it was missing previously

Reply:
Added in 60383eb49b0d44c507b304a2f160279d6d89ff6d